### PR TITLE
gitAndTools.ghr: init at 0.13.0

### PR DESCRIPTION
**Pull request by IvarWithoutBones**
###### Motivation for this change
Tests didn't seem to work, so I've added a simple check if the binary works.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/ghr/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  installCheckPhase = ''
+    $out/bin/ghr --version
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/tcnksm/ghr";
+    description = "Upload multiple artifacts to GitHub Release in parallel";
+    license = licenses.mit;
+    maintainers = [ maintainers.ivar ];
+    platforms = platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/ghr/default.nix:29]:
```suggestion
```

Already set by buildGoModule.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107660` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gitAndTools.ghr</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107660` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gitAndTools.ghr</li>
  </ul>
</details>

File: pkgs/applications/version-management/git-and-tools/ghr/default.nix (hunk 1 above)
Comment (IvarWithoutBones) [reply pkgs/applications/version-management/git-and-tools/ghr/default.nix]: Fixed, thanks :)

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107660` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gitAndTools.ghr</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107660` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gitAndTools.ghr</li>
  </ul>
</details>

**Pull request merged**